A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -3320,6 +3326,62 @@ class FMCMainDisplay extends BaseAirliners {
     isAltitudeManaged() {
         return SimVar.GetSimVarValue("AUTOPILOT ALTITUDE SLOT INDEX", "number") === 2;
     }
+
+    getTransitionAltitude(icao, phase) {
```

Comment:
As you've changed the behaviour here, a new name might be more appopriate:

```suggestion
    updateTransitionAltitude(icao, phase) {
```